lib/vf_actions/src/builtins.rs

```diff
@@ -7,52 +7,52 @@
  *
  * @see https://github.com/valueflows/valueflows/issues/487#issuecomment-482161938
  */
-use super::{
-    Action,
-    ActionEffect,
-    ProcessType,
-};
+use super::{Action, ActionEffect, ProcessType};
 
 // setup for core actions as in-memory statics
 
 macro_rules! generate_builtin_actions {
-    ($key: expr; $( $a:ident => $e:expr, $f:expr, $g:expr );*) => {
+    ($key: expr; $( $a:ident => $e:expr, $f:expr, $g:expr, $h:expr );*) => {
         match &str::replace($key, "-", "_")[..] {
             $(
                 stringify!($a) => Some(Action {
                     id: str::replace(stringify!($a), "_", "-"),
                     label: str::replace(stringify!($a), "_", "-"),
                     resource_effect: $e,
-                    input_output: $f,
-                    pairs_with: stringify!($g).to_string(),
+                    onhand_effect: $f,
+                    input_output: $g,
+                    pairs_with: stringify!($h).to_string(),
                 })
             ),*,
             _ => None,
         }
     }
 }
 
+// The first ActionEffect here is what is called the 'resource_effect' or
+// the 'accounting effect'. The second ActionEffect define the 'onhand_effect'.
+// They are separate because they can differ from each other for a given action.
```

Comment:
change to something like, "they are distinct because some actions mark a resource as 'in use'; ie. it's not 'on hand' anymore, but it hasn't been removed from inventory for accounting purposes and is still 'owned' by the same Agent(s)".
...Or ask @fosterlynn if she has a better explanation of the difference ;)

Reply:
if we want a deeper explanation maybe we can just provide that as a link to a reference instead of all this

Reply:
>...Or ask @fosterlynn if she has a better explanation of the difference ;)

That's basically right. :)  And I think better than what is there, because "resource effect" is confusing there.

>if we want a deeper explanation maybe we can just provide that as a link

I didn't find anything very succinct.  There is this page https://www.valueflo.ws/specification/event-resource/.  Or maybe the actions page https://www.valueflo.ws/concepts/actions/#action-behaviors.  Maybe I should write something in the latter page.

Reply:
"Maybe I should write something in the latter page."
I think there should definitely be something that equates the onhandEffect property of actions to the `onhandQuantity` of Resources. Of course it's obvious once you see it, and the wording conveys a connection, but it would be perfect for it to be explicit.

Reply:
""resource effect" is confusing there."
I agree. Why DO we have it called resourceEffect there?

Reply:
Right, resource effect can affect accounting and/or onhand quantities, depending on the action.  So it should be out of that statement.

Reply:
well, we don't currently have something in the code called "accounting effect", so should we replace "resource effect" with that? I'd like to if possible

Reply:
my confusion was merited. it is called 
resourceEffect in vf-graphql as of now: https://lab.allmende.io/valueflows/vf-schemas/vf-graphql/-/blob/sprout/lib/schemas/action.gql#L26
I am renaming it internally to accounting_effect, but aliasing it to `resourceEffect` when an action gets passed to the client (graphql) so that it doesn't error. 
We should fix that in vf-graphql?

Reply:
I created an issue here: https://lab.allmende.io/valueflows/vf-schemas/vf-graphql/-/issues/103

Reply:
Yes, graphql and VF itself.  A breaking change, but ah well, this isn't the worst time for that.